tests/tests-api.php

```diff
@@ -299,10 +299,10 @@ public function test_get_product_stats() {
 		$this->assertArrayHasKey( 'sales', $out['products'][0]['stats']['monthly_average'] );
 		$this->assertArrayHasKey( 'earnings', $out['products'][0]['stats']['monthly_average'] );
 
-		$this->assertEquals( '60', $out['products'][0]['stats']['total']['sales'] );
-		$this->assertEquals( '229.430000', $out['products'][0]['stats']['total']['earnings'] );
-		$this->assertEquals( '60', $out['products'][0]['stats']['monthly_average']['sales'] );
-		$this->assertEquals( '229.430000', $out['products'][0]['stats']['monthly_average']['earnings'] );
+		$this->assertEquals( '1', $out['products'][0]['stats']['total']['sales'] );
```

Comment:
I do not know where the original values came from for these--from what I see in this class, there's just one payment to evaluate and the product in question has one sale at `100.00`. But that's so different from the `229.430000` number that we maybe need to look into this more.